PocketKit/Sources/PocketKit/MyList/ItemsList/SavedItemsList/SavedItem+ItemsListItem.swift

```diff
@@ -8,23 +8,23 @@ extension SavedItem: ItemsListItem {
     }
 
     var domain: String? {
-        item?.domain
+        item.domain
     }
 
     var topImageURL: URL? {
-        item?.topImageURL
+        item.topImageURL
```

Comment:
not for this pr, but we should clean up our display logic for item and savedItem